src/main.js

```diff
@@ -41,9 +46,26 @@ function saveTask(title) {
   localStorage.setItem("tasks", tasks);
 }
 
+function pendingTasks(){

```

Comment:
You could name this `updatePendingTasks` to be more clear

Reply:
done